polygeist/tools/cgeist/Lib/CodeGenTypes.cc

```diff
@@ -38,10 +38,10 @@ using namespace mlir;
 using namespace llvm;
 
 static cl::opt<bool>
-    memRefFullRank("memref-fullrank", cl::init(false),
+    MemRefFullRank("memref-fullrank", cl::init(false),
                    cl::desc("Get the full rank of the memref."));
 
-static cl::opt<bool> memRefABI("memref-abi", cl::init(true),
+static cl::opt<bool> MemRefAbi("memref-abi", cl::init(true),
```

Comment:
I'd keepo `MemRefABI`, as "ABI" is not a regular word.

Reply:
ok